predicators/ground_truth_models/kitchen/options.py

```diff
@@ -33,111 +28,133 @@ def get_options(cls, env_name: str, types: Dict[str, Type],
-            assert "MoveTo" in op.name or "Push" in op.name
-            val = "move_delta_ee_pose"
-            if isinstance(primitive_name_to_action_idx[val], int):
-                params_space = Box(-np.ones(1) * 5, np.ones(1) * 5)
-            else:
-                n = len(primitive_name_to_action_idx[val])
-                params_space = Box(-np.ones(n) * 5, np.ones(n) * 5)
-            obj_types = [param.type for param in op.parameters]
-            options.add(
-                utils.ParameterizedOption(
-                    name=op.name.lower() + "_option",
-                    types=obj_types,
-                    params_space=params_space,
-                    policy=cls._create_policy(name=val),
-                    initiable=lambda _1, _2, _3, _4: True,
-                    terminal=cls._create_terminal(name=val, operator=op)))
-        return options
 
-    @classmethod
-    def _create_policy(cls, name: str) -> ParameterizedPolicy:
-
-        def policy(state: State, memory: Dict, objects: Sequence[Object],
-                   params: Array) -> Action:
-            del memory  # unused.
-            if name.lower() == "move_delta_ee_pose":
-                assert len(params) == 3
-                if len(objects) == 2:
-                    gripper, _ = objects
-                else:
-                    assert len(objects) == 3
-                    gripper, _, _ = objects
-                assert gripper.name == "gripper"
-                gx = state.get(gripper, "x")
-                gy = state.get(gripper, "y")
-                gz = state.get(gripper, "z")
-                dx = params[0] - gx
-                dy = params[1] - gy
-                dz = params[2] - gz
-                primitive_params = np.array([dx, dy, dz],
-                                            dtype=np.float32).clip(-0.1, 0.1)
+        max_delta_mag = 0.1  # don't move more than this per step
+
+        # MoveTo
+        def _MoveTo_initiable(state: State, memory: Dict,
+                              objects: Sequence[Object],
+                              params: Array) -> bool:
+            # Store the target pose.
+            _, obj = objects
+            ox = state.get(obj, "x")
+            oy = state.get(obj, "y")
+            oz = state.get(obj, "z")
+            target_pose = params + (ox, oy, oz)
+            memory["target_pose"] = target_pose
+            memory["reset_count"] = 8
+            return True
+
+        def _MoveTo_policy(state: State, memory: Dict,
+                           objects: Sequence[Object], params: Array) -> Action:
+            del params  # unused
+            gripper = objects[0]
+            gx = state.get(gripper, "x")
+            gy = state.get(gripper, "y")
+            gz = state.get(gripper, "z")
+            if memory["reset_count"] > 0:
+                arr = primitive_and_params_to_primitive_action(
+                    "move_backward", [1.0])
+                memory["reset_count"] -= 1
```

Comment:
could you add a comment or something explaining why there are (8) steps of moving backward here?